suite-native/module-send/src/screens/SendFeesScreen.tsx

```diff
@@ -18,23 +17,7 @@ export const SendFeesScreen = ({
         selectAccountByKey(state, accountKey),
     );
 
-    useEffect(() => {
-        // Subscribe to blocks for Solana, since they are not fetched globally
-        // this is needed for correct Solana fee estimation
-        if (account && account.networkType === 'solana') {
-            TrezorConnect.blockchainSubscribe({
-                coin: account.symbol,
-                blocks: true,
-            });
-
-            return () => {
-                TrezorConnect.blockchainUnsubscribe({
-                    coin: account.symbol,
-                    blocks: true,
-                });
-            };
-        }
-    }, [account]);
+    useSubscribeForSolanaBlockUpdates(account);
```

Comment:
SendOutputsScreen is not unmounted when SendFeesScreen is active, so the subscription is already active here, is that intentional? I guess it's not a problem and probably a bit safer to subscribe on both screens, but if you remove it from here, it should still works the same.

Reply:
@PeKne Mentioned the same thing. The intention was to make it work when coming to fee screen via different flow (trade), without specifying txn outputs.

I'll revisit this once more with other things we discussed

Reply:
changed in db7b7ffbcf